cmd/capibmadm/cmd/powervs/image/import.go

```diff
@@ -31,9 +31,10 @@ import (
 
 	logf "sigs.k8s.io/cluster-api/cmd/clusterctl/log"
 
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/iam"
 	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/powervs"
 	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/options"
-	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/utils"
+	pkgUtils "sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/cloud/services/utils"
```

Comment:
this can be just utils if no other utils package used!
```suggestion
	"sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/cloud/services/utils"
```

Reply:
Addressed 👍🏼